pusher.gemspec

```diff
@@ -1,7 +1,13 @@
 # -*- encoding: utf-8 -*-
+
+# Copied from https://github.com/bundler/bundler/blob/524add155d6b90c679db21b03f0bd9f877f21ab0/bundler.gemspec#L3-L5
+# but made less cryptic
+$LOAD_PATH.unshift(File.expand_path('../lib/', __FILE__))
+require "pusher/version"
```

Comment:
Or load the file directly. Ruby 1.9+ resolves the file to the absolute file anyways.

```ruby
require File.expand_path('../lib/pusher/version', __FILE__)
```